extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -97,86 +97,79 @@ const renderSystemLayerElement = (element, children) => (
     <div class="i-amphtml-story-has-new-page-notification-container">
       <div class="i-amphtml-story-has-new-page-text-wrapper">
         <span class="i-amphtml-story-has-new-page-circle-icon" />
-        <div class="i-amphtml-story-has-new-page-text">
-          {localize(
-            element,
-            LocalizedStringId_Enum.AMP_STORY_HAS_NEW_PAGE_TEXT
-          )}
-        </div>
+        <div class="i-amphtml-story-has-new-page-text"></div>
       </div>
     </div>
     <div class="i-amphtml-story-system-layer-buttons">
       <div
         role="button"
         class={INFO_CLASS + ' i-amphtml-story-button'}
-        aria-label={localize(
-          element,
-          LocalizedStringId_Enum.AMP_STORY_INFO_BUTTON_LABEL
-        )}
+        data-aria-label={LocalizedStringId_Enum.AMP_STORY_INFO_BUTTON_LABEL}
```

Comment:
This is valid AMP so publishers could potentially be using this attribute, or add some and mess with our implementation.
Also, it should be easier to recognize localization keys in our DOM.

You can for example use `i-amphtml-i18n-aria-label`

Reply:
I see the point, so you want to have a method that takes the template and looks for these "inner-text" or "aria-label" attributes and fills them appropriately, and we need `i-amphtml` attributes because this could happen in a light dom (because in this instance it wouldn't matter, it's a shadow dom)?